StRoot/StBFChain/doc/StBFChain.cxx_doc

```diff
@@ -1271,6 +1271,7 @@ Bfc_st BFC[] = {
   {"eemcDb" ,"eeDb" ,"","db,EEmcUtil", "StEEmcDbMaker","StEEmcDbMaker","Load EEmcDbMaker",kFALSE},
```

Comment:
Aren't the files in the "doc" directly are produced as a build artifact?

Reply:
Right, these files should be excluded from the PR.

Reply:
I see you approved the full PR, including these changes.... so I'll withhold my approval until this is resolved.

Reply:
Yeah, I probably should have just submitted mine as a comment instead of approval. One of many complications in life with git. Sorry.